tests/integration/test_fim/test_registry/test_registry_report_changes/test_disk_quota/test_registry_disk_quota_values.py

```diff
@@ -71,60 +73,62 @@
 
 # Variables
 
-key = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\key1"
-sub_key_2 = "SOFTWARE\\key2"
+key = WINDOWS_HKEY_LOCAL_MACHINE
+sub_key_1 = MONITORED_KEY
+sub_key_2 = MONITORED_KEY_2
 
-test_regs = [os.path.join(key, sub_key_1),
-             os.path.join(key, sub_key_2)]
-
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_regs = [os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)]
 reg1, reg2 = test_regs
-size_limit_configured = 10 * 1024
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+size_limit_configured = SIZE_LIMIT_CONFIGURED_VALUE
 
 # Configurations
 
-p, m = generate_params(modes=['scheduled'], extra_params={'WINDOWS_REGISTRY_1': reg1,
-                                                          'WINDOWS_REGISTRY_2': reg2,
-                                                          'FILE_SIZE_ENABLED': 'no',
-                                                          'FILE_SIZE_LIMIT': '10KB',
-                                                          'DISK_QUOTA_ENABLED': 'yes',
-                                                          'DISK_QUOTA_LIMIT': '1KB'})
-
-configurations_path = os.path.join(test_data_path, 'wazuh_registry_report_changes_limits_quota.yaml')
+p, m = generate_params(modes=["scheduled"], extra_params={
```

Comment:
changes the variables in order to representative
```suggestion
conf_params, conf_metadata = generate_params(modes=["scheduled"], extra_params={
```